### PR TITLE
Fix/discard unclassified mags

### PR DESCRIPTION
**Pull request by KateSakharova**
I removed unclassified genomes from tsv file for uploader. 
Script also produces file with a list of unclassified genomes. Not sure that we need to create a folder with those. They can be easily taken using genomes folder and that output file.

### CODE REVIEW

File: bin/tsv_for_genome_upload.py [hunk 1]
```diff
@@ -363,18 +372,27 @@ def process_tax_file(self, filename, type):
                     line = line.strip().split('\t')
                     if type == 'euks':
                         lineage[line[0]] = line[3]
+                        if line[3] == 'Unclassified':
```
Comment (Ge94) [inline bin/tsv_for_genome_upload.py:375]: Does the eukaryotic taxonomy annotator also return "Unclassified"? Or is there another equivalent?
(I thought only GTDB was returning Unclassified...)

Comment (Ge94) [reply bin/tsv_for_genome_upload.py:375]: In other words - should we check for an equivalent in the eukaryotic annotations?

**Review by Ge94 — COMMENTED** [review]
Everything looks good, as long as you can confirm the eukaryotic annotation comment, then ready to go

File: bin/tsv_for_genome_upload.py (hunk 1 above)
Comment (KateSakharova) [reply bin/tsv_for_genome_upload.py:375]: good catch, I'm not aware of it. Let's assume they don't have unclassified (until we meet them at some point)

**Pull request merged**